keyboards/canary/canary60rgb/keymaps/via/keymap.c

```diff
@@ -21,20 +21,20 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
         KC_TAB,         KC_Q,    KC_W,            KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC,         KC_RBRC, KC_BSLS,
         KC_CAPS,        KC_A,    KC_S,            KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,                  KC_ENT,
         KC_LSFT,                 KC_Z,            KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  RSFT_T(KC_SLSH), KC_UP,   LT(2, KC_DEL),
-        KC_LCTL,        KC_LGUI, LM(1, MOD_LALT),                            KC_SPC,                    KC_RALT, MO(1),   KC_LEFT,         KC_DOWN, KC_RGHT
+        KC_LCTL,        KC_LGUI, KC_LALT,                                    KC_SPC,                    KC_RALT, MO(1),   KC_LEFT,         KC_DOWN, KC_RGHT
     ),
     [1] = LAYOUT(
         _______,   KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_DEL,
         _______, _______, KC_UP,   _______, _______, _______, KC_CALC, _______, KC_INS,  _______, KC_PSCR, KC_SLCK, KC_PAUS, RESET,
         _______, KC_LEFT, KC_DOWN, KC_RGHT, _______, _______, _______, _______, _______, _______, KC_HOME, KC_PGUP,          EEP_RST,
         _______,          _______, _______, _______, _______, _______, NK_TOGG, _______, _______, KC_END,  KC_PGDN, KC_VOLU, KC_MUTE,
-        _______, _______, TO(3),                            TO(4),                     _______, _______, KC_MPRV, KC_VOLD, KC_MNXT
+        _______, _______, TO(3),                              _______,                   _______, _______, KC_MPRV, KC_VOLD, KC_MNXT
     ),
     [2] = LAYOUT(
         _______, KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_DEL,
         _______, RGB_TOG, RGB_MOD, RGB_HUI, RGB_HUD, RGB_SAI, RGB_SAD, RGB_VAI, RGB_VAD, _______, _______, _______, _______, RESET,
         _______, _______, _______, _______, _______, _______, _______, _______, RGB_SPI, RGB_SPD, _______, _______,          EEP_RST,
-        _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+        _______,          BL_TOGG, BL_STEP, BL_INC,  BL_DEC,  BL_ON,   BL_OFF, _______, _______, _______, _______, _______, _______,
```

Comment:
Please use the RGB keycodes, not Backlight.

Reply:
Hi @fauxpark, I already used RGB keycode to control RGB matrix per key. We added RGB lighting WS2812 for this PCB and wanted to control RGB lighting and matrix separately. Therefore we wrote custom code to use Backlight keycodes to control RGB lighting.
If user want to used Backlight keycode, they would define their own function for these keycode
process_record_user(keycode, record))
I follow the guidance for keycode in this link:
https://beta.docs.qmk.fm/using-qmk/hardware-features/lighting/feature_rgblight

Please consider my changes.

Thanks.

Reply:
That page says nothing about using Backlight keycodes to control RGB. IMO repurposing these keycodes only perpetuates the confusion between the different lighting features.

Eventaully the keycodes for RGBLight/RGB Matrix will be separated so they can be controlled independently, but for now you just have to create keyboard-level custom keycodes to control the other one.

Reply:
Hi @fauxpark,
Ok, I will not re-purpose Backlight keycode for RGBLight controlling.
Those code are removed.
Please consider my changes.
Thanks.